src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -48,7 +54,18 @@ public void Load(NodeSettings nodeSettings)
             var apiHost = config.GetOrDefault("apiuri", DefaultApiHost);
             Uri apiUri = new Uri(apiHost);
 
-            var apiPort = config.GetOrDefault("apiport", nodeSettings.Network.IsBitcoin() ? DefaultBitcoinApiPort : DefaultStratisApiPort);
+            // Find out which port should be used for the API.
+            int port;
+            if (nodeSettings.Network.IsBitcoin())
```

Comment:
I believe we agreed this will go in Network right? so we don't need to make such checks as `Network.IsBitcoin()`

Reply:
These are only when the API feature is up. At the moment the features have their own settings, but when we redesign Network and its settings we can certainly review that.